Documentation/Wired.md

```diff
@@ -0,0 +1,228 @@
+
+# Wired Assembly
+
+*How to build a cheap **Hotdog**.*
+
+<br>
+
+If you have access to a 3D printer and use a cheap <br>
```

Comment:
The benefit is their functionality, additional spacing.

GitHub is 'responsive' in that it does some responsive things,
not that it does everything / does everything well.

Vertical separation is quite impactful.

Reply:
It is conventional for text documents to use the available horizontal space (see any news site, blog, Wikipedia etc.), and that is what most people are used to.
If you don't like long lines, then please at least add the line breaks along logical boundaries (e. g. between sentences).

Reply:
> see any news site, blog

Those are examples that usually don't do this as they 
understand, as newspapers / books / papers /.. do, 
that the human perception limit is about 60 characters.

I usually go with about 60 - 80 as a compromise.

> If you don't like long lines, then please at least add the line breaks along logical boundaries (e. g. between sentences).

Well since GitHub isn't really responsive, it doesn't have 
the required functionality of the `justify` alignment type 
which is required, if you want to cut sentences at length 
wise random positions.

Reply:
> so breaking along a given character limit isn't that meaningful IMHO.

I know, I try to make lines visually have the same 
length, as this also helps readability, however this 
usually requires me to reword sentences multiple 
times until I find the 'perfect' wording, which I didn't 
do much in this PR.

In intial PRs of bigger repos I usually first 
go for basic groundwork ( which is still a lot 
of work ) as they might very well get denied 
outright, sometimes without even a reason.

Reply:
I appreciate the effort, but please keep in mind, that now I'll have to reindent the lines each time I edit these documents, which is not necessarily something I'll enjoy doing. Which is why I have a strong preference to keep the default indentation, or at least to have it split into manageable chunks (such as sentences), and not something that will cascade through 10 lines.

Reply:
> I appreciate the effort, but please keep in mind, that now I'll have to reindent the lines each time I edit these documents, which is not necessarily something I'll enjoy doing. 

Huh, any modern editor should follow the indentation 
level of the previous line / line with content though..

I'm a bit confused how you came to indentation level
as this `request changes` conversation was about line
length..

In case you meant line length and are concerned about
having to keep the visual length equal from section to
section then just don't.

Having many small blocks with slightly different block
widths looks fine as long as the block / sentence is 
broken normally.

Not that you have to do it properly yourself anyways,
you could just half hazardly break them wherever and
have them be cleaned up by me in some future PR.

Reply:
I mean line width - sorry for the confusion.
Fine. Keep the line breaks, but please use the proper Markdown syntax, which is two spaces. Not having any of this "haha readme go `<br>`" menace.
As an aside, most browsers (Chrome, Firefox, Edge) come with a reader mode that does pretty much the same thing, so I still find line breaking by hand redundant.

Reply:
> I mean line width - sorry for the confusion.

No problem~

> but please use the proper Markdown syntax, which is two spaces

Are you referring to vertical spacing,
maybe I'm just stupid but I'm not sure
what you mean.

> As an aside, most browsers (Chrome, Firefox, Edge) come with a reader mode that does pretty much the same thing, so I still find line breaking by hand redundant.

Well most people don't use it.

If it's not automatically used
not many people will use it to
begin with..

Btw, thankfully Edge is dead~

Reply:
https://www.markdownguide.org/basic-syntax#line-breaks

Reply:
Mhmhmh, very interesting indeed.

Tried it out, very cool~

<br>

Though since markdown doesn't highlight trailing spaces 
and many linting tools automatically remove them I'm not 
sure if I'll prefer them in the future.

<br>

***I'll adjust the PR then~***